### ISSUE TITLE
Encoder direction change = "Too many encoders, FAIL!"

### ISSUE THREAD

**Issue by KSSS1** [madhephaestus/ESP32Encoder#45]
Hi,

I want to change encoder direction. I know it is possible to change this in software(SETUP) but i want to change on the fly in the void loop by attaching the pins reversed  "encoderX.attachHalfQuad(2, 3)" to  encoderX.attachHalfQuad(3, 2) When a button is pressed. This works **BUT**
when you do this a couple times in the loop then it gives on the serial monitor this **error** "Too many encoders, FAIL!" So when this error occurs there is no possibility to change the encoder directions anymore.

Is there a possibility to erase the ATTACHED encoders or to reset the encoder attached count? a funtion like "encoder_detach"? 

Thanks!

**Comment by madhephaestus** [comment]
That is a use case i never thought of. Sure, go ahead and submit a PR for an encoder.detach() method and ill take a look :)

**Comment by madhephaestus** [comment]
This feature is now availible in 0.7.0
